pkg/render/layer.go

```diff
@@ -79,30 +76,24 @@ func getRenderValues(inputs ValuesInputs) (interface{}, error) {
 
 	// TODO: get rid of this and just use the ValuesInputs type
 	type manifestRenderValues struct {
-		Name               string
-		InstallNamespace   string
-		FlavorName         string
-		MeshRef            core.ResourceRef
-		SuperglooNamespace string
-
-		Supergloo SuperglooInfo
+		Name             string
+		InstallNamespace string
+		//FlavorName       string
```

Comment:
Why commented out?

Reply:
flavor name isn't needed for rendering custom values, removing.